backend/app/db/crud/document.py

```diff
@@ -732,7 +732,7 @@ def create_document_relationship(
         # The call to db.commit() must exist outside the nested transaction
         db.commit()
     except IntegrityError:
-        # ensure idemopotency
+        # ensure impotency
```

Comment:
oops

Reply:
don't do that!!

Reply:
Possibly one of my favourite typos I've seen in a PR